internal/retryafter_test.go

```diff
@@ -32,7 +33,6 @@ func assertDuration(t *testing.T, duration OptionalDuration, expected time.Durat
 
 func TestExtractRetryAfterHeaderDelaySeconds(t *testing.T) {
 	// Generate random n > 0 int
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Why the deletion?

Reply:
Seed is deprecated: https://pkg.go.dev/math/rand#Seed

Reply:
I see. The new behavior is:
>If Seed is not called, the generator is seeded randomly at program startup.

That's all we need, so good to delete it.